src/Pos/Wallet/Web/Api.hs

```diff
@@ -104,46 +136,47 @@ type WalletApi =
      :> "txs"
      :> "payments"
      :> Capture "passphrase" CPassPhrase
-     :> Capture "from" CAddress
-     :> Capture "to" CAddress
+     :> Capture "from" CWalletAddress
+     :> Capture "to" (CAddress Acc)
      :> Capture "amount" Coin
-     :> Post '[JSON] (Either WalletError CTx)
+     :> Post '[JSON] (Either WalletError [CTx])
     :<|>
     -- TODO: for now we only support one2one sending. We should extend this to support many2many
      "api"
      :> "txs"
      :> "payments"
      :> Capture "passphrase" CPassPhrase
-     :> Capture "from" CAddress
-     :> Capture "to" CAddress
+     :> Capture "from" CWalletAddress
+     :> Capture "to" (CAddress Acc)
      :> Capture "amount" Coin
      :> Capture "currency" CCurrency
      :> Capture "title" Text
      :> Capture "description" Text
-     :> Post '[JSON] (Either WalletError CTx)
+     :> Post '[JSON] (Either WalletError [CTx])
```

Comment:
AFAIU we can spend money from several accounts simultaneously here, single transaction relates to single source address (because it should be signed with some secret key)